src/remote_gym/remote_environment_management.py

```diff
@@ -11,12 +18,15 @@
     dm_env_rpc_pb2_grpc,
     spec_manager,
     tensor_spec_utils,
+    tensor_utils,
 )
 from google.rpc import code_pb2, status_pb2
 
+from remote_gym.repo_manager import RepoManager
+
 
 def start_as_remote_environment(
```

Comment:
I think at this point we might want to rename this method to `create_environment_in_server` or something like that.